cmd/shared-resources-operator/main.go

```diff
@@ -49,12 +54,30 @@ func NewOperatorCommand() *cobra.Command {
 	ctrlCmd := controllercmd.NewControllerCommandConfig(
 		"shared-resources-operator",
 		version.Get(),
-		operator.RunOperator,
+		runOperatorWithKubeconfig,
 	).NewCommandWithContext(context.TODO()) //TODO cmd.Context()) came back with panic: cannot create context from nil parent
 	ctrlCmd.Use = "start"
 	ctrlCmd.Short = "Start the Projected Shared Resources Operator"
+	kubeconfig = ctrlCmd.Flags().String("kubeconfig", "", "Path to kubeconfig file.  If not provided, configured service account will be used.")
```

Comment:
per https://github.com/openshift/csi-driver-shared-resource-operator/pull/70#issuecomment-1431900805 looks like this flag already exists

I'll check out this PR, see if I can tweak things to access

Reply:
https://github.com/openshift/csi-driver-shared-resource-operator/pull/71